packages/data-stores/src/wpcom-plans-ui/selectors.ts

```diff
@@ -1,3 +1,7 @@
 import type { State } from './reducer';
 
 export const isDomainUpsellDialogShown = ( state: State ) => !! state.showDomainUpsellDialog;
+export const getStorageAddOnForPlan = ( state: State ) => ( plan: string ) => {
```

Comment:
`plan` -> `planSlug` applies for here too. Aso a proper type.

Reply:
Good point -- made the change in [2b80eda](https://github.com/Automattic/wp-calypso/pull/80158/commits/2b80eda51d7c6308773462ee5b0a472f28b845aa)